samples/CallWithChat/.babelrc.js

```diff
@@ -1,5 +1,15 @@
 const commonConfig = require('../../common/config/babel/.babelrc.js');
 
 module.exports = {
-  ...commonConfig
+  ...commonConfig,
+  /**
+   * retainLines is needed to maintain the lines of the comments in the output to preprocess. This will cause in the output
+   * the comments assigned to their original position in the file so any linter options will be maintained on their respective node.
+   *
+   * this is caused by the comments being both trailing and leading to the nodes either above or below it, regardless
+   * of which node it is touching on the next line.
+   *
+   * link to babel docs on this: https://babeljs.io/docs/en/options#retainlines
```

Comment:
<3

The documentation says it is best effort, but that is good enough for us as we're using it for source-to-source transformation that we can (to some degree) eyeball as necessary.